### PR TITLE
[REF] account: remove unused actions

### PR DESCRIPTION
**Pull request by william-andre**
Task [2201948](https://www.odoo.com/web#id=2201948&action=333&active_id=967&model=project.task&view_type=form&cids=1&menu_id=4720)

* These actions were either unused or only used in enterprise modules.
The first ones have been deleted, and the second ones have been move to
the corresponding enterprise modules.

* `account.fiscal.year`: the whole model has been moved to enterprise

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/account/views/product_view.xml [hunk 1]
```diff
@@ -49,5 +49,7 @@
                 </group>
             </field>
         </record>
+
+        <menuitem id="menu_product_product_categories" action="product.product_category_action_form" parent="account.account_account_menu" sequence="3"/>
```
Comment (qdp-odoo) [inline addons/account/views/product_view.xml:53]: needs to go in addons/account/views/account_menuitem.xml now

**Comment by qdp-odoo** [comment]
@william-andre please let me know when this branch is ready

**Comment by william-andre** [comment]
@qdp-odoo Rebased and migration (explicit removal of records) written

**Comment by qdp-odoo** [comment]
@robodoo r+

**Comment by qdp-odoo** [comment]
it's red in here

**Comment by qdp-odoo** [comment]
@robodoo r+

**Comment by smetl** [comment]
@william-andre this branch is red due to the migration scripts

**Comment by william-andre** [comment]
Seems like a bad rebase, I just redid the whole thing, and tested in local without any issue

**Comment by william-andre** [comment]
@smetl The migration tests passed on both enterprise and community

**Comment by qdp-odoo** [comment]
@robodoo r+

**Pull request closed without merging**